.buildkite/hooks/pre-command

```diff
@@ -62,13 +62,34 @@ if [[ "$BUILDKITE_PIPELINE_SLUG" == "beats" || "$BUILDKITE_PIPELINE_SLUG" == "fi
   fi
 fi
 
-if [[ "$BUILDKITE_PIPELINE_SLUG" == "beats-metricbeat" || "$BUILDKITE_PIPELINE_SLUG" == "beats-libbeat" || "$BUILDKITE_PIPELINE_SLUG" == "beats-packetbeat" || "$BUILDKITE_PIPELINE_SLUG" == "beats-winlogbeat" || "$BUILDKITE_PIPELINE_SLUG" == "beats-xpack-libbeat" || "$BUILDKITE_PIPELINE_SLUG" == "beats-xpack-metricbeat" || "$BUILDKITE_PIPELINE_SLUG" == "beats-xpack-packetbeat" || "$BUILDKITE_PIPELINE_SLUG" == "beats-xpack-winlogbeat" || "$BUILDKITE_PIPELINE_SLUG" == "beats-xpack-dockerlogbeat" || "$BUILDKITE_PIPELINE_SLUG" == "beats-xpack-auditbeat" || "$BUILDKITE_PIPELINE_SLUG" == "beats-xpack-filebeat" || "$BUILDKITE_PIPELINE_SLUG" == "beats-xpack-heartbeat" || "$BUILDKITE_PIPELINE_SLUG" == "beats-xpack-osquerybeat" ]]; then
-  if [[ "${BUILDKITE_COMMAND}" =~ ^buildkite-agent ]]; then
-    echo "Skipped pre-command when running the Upload pipeline"
-    exit 0
+ENABLED_BEATS_PIPELINES_SLUGS=(
+  "auditbeat"
+  "filebeat"
+  "beats-metricbeat"
+  "beats-libbeat"
+  "beats-packetbeat"
+  "beats-winlogbeat"
+  "beats-winlogbeat"
+  "beats-xpack-libbeat"
+  "beats-xpack-metricbeat"
+  "beats-xpack-packetbeat"
+  "beats-xpack-winlogbeat"
+  "beats-xpack-dockerlogbeat"
+  "beats-xpack-auditbeat"
+  "beats-xpack-filebeat" 
+  "beats-xpack-heartbeat" 
+  "beats-xpack-osquerybeat"
+  )
+
+for slug in "${ENABLED_BEATS_PIPELINES_SLUGS[@]}"; do
+  if [[ "$BUILDKITE_PIPELINE_SLUG" == "$slug" ]]; then
+    if [[ "${BUILDKITE_COMMAND}" =~ ^buildkite-agent ]]; then
```

Comment:
**nit**: this condition could be added earlier so no need to iterate through the loop if `"${BUILDKITE_COMMAND}" =~ ^buildkite-agent`

```
if [[ ! "${BUILDKITE_COMMAND}" =~ ^buildkite-agent ]]; then
  for slug in "${ENABLED_BEATS_PIPELINES_SLUGS[@]}"; do
    if [[ "$BUILDKITE_PIPELINE_SLUG" == "$slug" ]]; then
      source .buildkite/scripts/setenv.sh
```

Reply:
I was going to do the same but probably it was done so for some reasons